### PR TITLE
chore(Bundle): rename the bonita bundle from

### PR DESCRIPTION
**Pull request by danila-m**
BonitaSubscription-7.x.y-tomcat into
BonitaSubscription-7.x.y

Relates to [BR-224](https://bonitasoft.atlassian.net/browse/BR-224)

### CODE REVIEW

File: md/bonita-as-windows-service.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Please verify the supported version from the Support page
 
 ![Tomcat home page](images/bonita-as-windows-service/tomcatHome.png)
 
-* **_Download_** the bundle BonitaSubscription-x.y.z-tomcat.zip (or BonitaCommunity-x.y.z-tomcat.zip for the community edition)
+* **_Download_** the bundle BonitaSubscription-x.y.z.zip (or BonitaCommunity-x.y.z-tomcat.zip for the community edition)
```
Comment (passga) [inline md/bonita-as-windows-service.md]: BonitaCommunity-x.y.z-tomcat.zip  -> it stay tomcat in BonitaCommunity zip name

**Comment by tbouffard** [comment]
Magnifico 👏

**Review by passga — COMMENTED** [review]
ok for me,  just a forgets on the name of BonitaCommunity zip file

**Review by educhastenier — CHANGES_REQUESTED** [review]
You're missing the RN update

**Comment by tnguyen1** [comment]
retest this please

File: md/bonita-as-windows-service.md (hunk 1 above)
Comment (educhastenier) [reply md/bonita-as-windows-service.md]: @danila-m ?

File: md/release-notes.md [hunk 2]
```diff
@@ -4,6 +4,11 @@
 **Note:** The 7.10 is currently work in progress (WIP). The 7.10.0 GA is planned on December 2019.
 :::
 
+## Global Changes
+### Renaming
+The Tomcat bundle packaging has changed. Previously called _Bonita{version}-7.x.y-tomcat_, it has been renamed _Bonita{version}-7.x.y_.
```
Comment (educhastenier) [inline md/release-notes.md]:
```suggestion
The Tomcat bundle packaging has changed. Previously called _Bonita{edition}-7.x.y-tomcat_, it has been renamed to _Bonita{edition}-7.x.y_.
```

**Review by educhastenier — APPROVED** [review]
Looks good to me :+1:

**Comment by tbouffard** [comment]
@educhastenier @danila-m can we merge this PR? everything looks good

**Pull request merged**